fr/lecons/nettoyer-ses-donnees-avec-openrefine.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Nettoyer ses données avec OpenRefine
+layout: lesson
+date: 2013-08-05
+translation_date: 2018-11-27
+authors:
+- Seth van Hooland
+- Ruben Verborgh
+- Max De Wilde
+reviewers:
+- Patrick Burns
+- Nora McGregor
+editors:
+- Adam Crymble
+translator:
+- Sybille Clochet
+translation-editor:
+- Sofia Papastamkou
+translation-reviewer:
+- François Dominic Laramée
+- Marie Puren
+difficulty: 2
+exclude_from_check:
```

Comment:
So this translation actually _does_ have a review ticket, unlike the old lesson, so please remove this entry from the YAML

Reply:
just done this and previous requested changes